ostelco-ios-client/Extensions/UIViewController+StartApplePay.swift

```diff
@@ -61,4 +63,21 @@ extension UIViewController {
             #endif
         }
     }
+
+    // Stripe (& Prime) expects amounts to be provided in currency's smallest unit.
+    // https://stripe.com/docs/currencies#zero-decimal
+    // https://github.com/stripe/stripe-ios/blob/v15.0.1/Stripe/NSDecimalNumber%2BStripe_Currency.m
+    func convertStripeToNormalCurrency(amount: Decimal, currency: String) -> NSDecimalNumber {
```

Comment:
is there a reason we can't just `import Stripe` here and use their implementation instead of copying it?

Reply:
I wasn't able to import that extension. Can you try?

Reply:
yep, hold plz....

Reply:
wait a minute, I'm gonna DM you, something's not quite adding up here

Reply:
OK it's not just Stripe + Prime, it's `PKPaymentSummaryItem` as well. Whee!